.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "current"
+      }
+    }]
+  ]
+}
```

Comment:
Your files should end with a new line. Please fix this in all instance of your file that has the same problem